imports/wasi_snapshot_preview1/poll_test.go

```diff
@@ -137,7 +124,7 @@ func Test_pollOneoff_Errors(t *testing.T) {
 			resultNevents: 512, // past out
 			expectedMem: []byte{
 				0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, // userdata
-				byte(expectedFdReadErr), 0x0, // errno is 16 bit
+				byte(wasip1.ErrnoSuccess), 0x0, // errno is 16 bit
```

Comment:
technically this is no longer an error and it is no longer returning unsupported or EBADF so maybe it makes sense to give the test case a different name.